core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -164,17 +175,17 @@ public String toString() {
       short mode = mSrcInode.getMode();
       return String.format(
```

Comment:
check if the client version is null. if yes, don't include it in such string

Reply:
If the `mClientVersion` variables is null, it will print `clientVersion=null`. i don't think there is a need to change this.